packages/labs/src/components/popover/popover2.tsx

```diff
@@ -260,7 +260,7 @@ export class Popover2 extends AbstractComponent<IPopover2Props, IPopover2State>
 
         const children = this.understandChildren();
         const targetTabIndex = this.props.openOnTargetFocus && this.isHoverInteractionKind() ? 0 : undefined;
-        const target = React.cloneElement(children.target,
+        const target: JSX.Element = React.cloneElement(children.target,
```

Comment:
This is necessary, otherwise we get:
```
error TS7022: 'target' implicitly has type 'any' because it does not have a type annotation and is referenced directly or indirectly in its own initializer.
```
Error started showing up after changing `Tooltip` to `Tooltip2` below